libs/coin-modules/coin-canton/src/network/node.ts

```diff
@@ -1,12 +1,46 @@
 import { SimulationError } from "../types/errors";
-import network from "@ledgerhq/live-network/network";
+import network, { LiveNetworkRequest } from "@ledgerhq/live-network/network";
 import { getEnv } from "@ledgerhq/live-env";
 import coinConfig from "../config";
 import { AccountInfoResponse, SubmitReponse } from "./types";
+import crypto from "crypto";
 
 const getNodeUrl = () => coinConfig.getCoinConfig().nodeUrl;
+const getNetworkType = () => coinConfig.getCoinConfig().networkType;
 
-// NOTE: add NODE_BOILERPLATE to libs/env/src/env.ts
+export const generateJWT = () => {
```

Comment:
I would add some note that this token is only correct for localnet.

Reply:
indeed, thanks